doc/design/vic-machine.md

```diff
@@ -179,6 +179,47 @@ Some of the elevated privilege operations could be delegated during self-provisi
 
 At this point install transitions to managing - reporting VCH status from initial install is the same as reporting that information for any VCH regardless of age.
 
+
+## Deleting - per vSphere target
+
+### Inputs
+
+1. vSphere SDK endpoint
+2. vSphere administrative credentials
+3. VCH identification
+ - VCH resource pool path in govc format, e.g. /ha-datacenter/vm/vch1
+
+### Actions
+
+1. Get VCH VM
+2. Read back VCH configuration from VM guestinfo
+3. Delete following resources based on VCH configuration.
+ - Container VMs managed by the VCH
+   - Contianer datastore paths and resource pool path configured during installation will be used here, to detect if the VM belongs to this VCH. If yes, these VMs will be removed.
+   - Container VMs will be removed even -force is not specified
+ - volumes managed by the VCH
+   - -force option is required to delete volumes together with VCH uninstallation.
+    - Volume datastore path (sample: ds://datastore1/volume/vch1) configured during installation will be used here, to detect if the volumes are created by this VCH.
+     - If volume directory (sample: ds://datastore1/volume/vch1) is created by vic-machine create, vic-machine delete should delete this directory as well (how to know if the directory is created by vic-machine create?).
+ - images managed by the VCH
+   - Images are shared by VCHs. Currently no reference or metadata to specify which image is referenced by which VCH, so for TP3, vic-machine delete will not touch images.
+ - vSphere networks created
+   - vic-machine create or VCH port-layer-server will create network, so vic-machine delete should delete any new networks. (Still need to confirm for how to get created networks)
+ - VCH specific metadata from vsphere objects if any
+   - Cause image will not be removed at this time, image metadata will not touched as well. But for container metadata, if any, will be deleted. (Need to finalize where it is)
+ - the resource pool and appliance VM
+
+### Samples
+```
+vic-machine delete 
+-target 192.168.1.1
```

Comment:
Looking at the vic-machine.md PR I've got in progress the target specifier is:
`<user:password>@fqdn/path/to/datacenter/`

This is because the datacenter is the top level namespace and is necessary to disambiguate all other resources. In the case of ESX there can be only one, but in general:
`-target=https://root@****:vcenter.fqdn/path/to/datacenter`

My apologies for the shifting of argument specifications - it's needed quite a few iterations to get to something sane(ish). I still need to revise the examples with the -FROM argument.